evaluation/human_evaluation/main.py

```diff
@@ -0,0 +1,116 @@
+import streamlit as st
+from dotenv import load_dotenv
+import os
+from typing import List
```

Comment:
technically you can use `list`, since we are using python>=3.10

Reply:
Yeah, well if people want to use just the frontend, maybe it's okay to make it backward compatible ? otherwise I can remove it

Reply:
I just want to make it consistent with the backend. Talk with Pal on what you guys prefer.

Reply:
I think there are some scripts, that run only on python 3.10+, so Pal will have to change the code for that, I can remove this to make it consistent. If that sounds okay ?